### PR TITLE
 Add script for using assume role

### PR DESCRIPTION
**Pull request by ESergiusz**

### CODE REVIEW

File: script_for_using_sts_assume_role.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# For use this script you need to install jq "sudo apt install jq"
+read -p 'Input token code: ' token_key
+read -p 'Input profile name: ' profile
+json=$(aws sts assume-role --role-arn arn:aws:iam::546240550610:role/Admins \
+--role-session-name tf \
+--serial-number arn:aws:iam::546240550610:mfa/$profile \
```
Comment (ggramal) [inline script_for_using_sts_assume_role.sh:7]: hmm so if i have mfa name like `arn:aws:iam::546240550610:mfa/test` the script will not work right :thinking:

File: script_for_using_sts_assume_role.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# For use this script you need to install jq "sudo apt install jq"
+read -p 'Input token code: ' token_key
+read -p 'Input profile name: ' profile
+json=$(aws sts assume-role --role-arn arn:aws:iam::546240550610:role/Admins \
```
Comment (ggramal) [inline script_for_using_sts_assume_role.sh:5]: can we get role arn from profile something like 
`aws configure  --profile dos14 get role_arn`

File: script_for_using_sts_assume_role.sh (hunk 1 above)
Comment (ESergiusz) [reply script_for_using_sts_assume_role.sh]: Made a serial-number request via aws configure --profile $profile get role_arn

File: script_for_using_sts_assume_role.sh (hunk 2 above)
Comment (ESergiusz) [reply script_for_using_sts_assume_role.sh]: It doesn't work for me via aws configure --profile $profile get role_arn ((

Comment (ggramal) [reply script_for_using_sts_assume_role.sh]: Ok can we somehow get the ID of profiles project?

Comment (ESergiusz) [reply script_for_using_sts_assume_role.sh]: Create a new version script.

**Review by ggramal — APPROVED** [review]
Nice

**Pull request merged**